packages/ra-core/src/Resource.js

```diff
@@ -115,7 +113,6 @@ export class Resource extends Component {
                 )}
                 {edit && (
                     <Route
-                        exact
```

Comment:
doesn't it mean that the route `/posts/12/show` will also render this route?

Reply:
No because we're in a switch and the show route will take precedence

Reply:
gotcha